client/src/js/components/bhCostCenterSelect/bhCostCenterSelect.js

```diff
@@ -37,8 +37,9 @@ function CostCenterSelectController(CostCenters, Notify) {
   }
 
   $ctrl.$onInit = () => {
+    $ctrl.required = !!($ctrl.required);
     $ctrl.label = $ctrl.label || 'COST_CENTER.TITLE';
-    $ctrl.costCenterId = +$ctrl.costCenterId;
+    $ctrl.costCenterId = $ctrl.costCenterId ? +$ctrl.costCenterId : null;
```

Comment:
What does the + do here?

Reply:
Tries to coerce a string into a number, I assume.  It's not very clean code. 😃